apps/arm/CMakeLists.txt

```diff
@@ -1,29 +1,26 @@
 add_executable(arm.${TARGET}-board.elf)
 target_sources(arm.${TARGET}-board.elf PRIVATE src/main.cpp)
+target_include_directories(arm.${TARGET}-board.elf PUBLIC include)
 target_link_libraries(arm.${TARGET}-board.elf
-        PUBLIC
-        PID
         PRIVATE
-        ClawController
-        CANBuffer
-        CANBus
-        DifferentialWristController
-        EncoderRelative_Quadrature
+#Control
+        OpenLoopController
+        PositionController
+        VelocityController
+        CurrentController
+        ActuatorControllerManager
+#Motors
         LimServo
-        LookupTable
-        Motor
-        uwrt-mars-rover-hw-bridge
-        )
+        DCMotor
+#Encoders
+        EncoderAEAT6012
+        EncoderMAE3
+        EncoderPololu37D
```

Comment:
Recommend naming this the encoder name instead of the motor name: `Encoder64CPR`

Reply:
hmm ig that just means 64 counts per revolution and there may be many encoders with 64 counts per revolution. if you think thats better than pololu37d i can change it

Reply:
Can we have a call abt this? msg me on discord when ur free. I think its much easier to sort this out over a call